src/screens/ContactScreen/index.js

```diff
@@ -383,7 +446,7 @@ const ContactScreen = ({navigation}) => {
         </View>
       )}
 
-      {isRecyclerViewShown && !isSearchMode && (
+      {isRecyclerViewShown && usersSearch.length <= 0 && !isLoadingSearchResult && (
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition to show the RecyclerListView has been updated to check if `usersSearch.length <= 0` and `!isLoadingSearchResult`. This makes sense as we don't want to show the list view when there are no users to display or when the search result is still loading.



<!-- This is an auto-generated comment by OSS CodeRabbit -->